src/pages/home/report/ReportActionItemParentAction.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import type {WithLocalizeProps} from '@components/withLocalize';
+import type {WindowDimensionsProps} from '@components/withWindowDimensions/types';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ReportActionsUtils from '@libs/ReportActionsUtils';
+import * as Report from '@userActions/Report';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type * as OnyxTypes from '@src/types/onyx';
+import AnimatedEmptyStateBackground from './AnimatedEmptyStateBackground';
+import ReportActionItem from './ReportActionItem';
+
+type ReportActionItemParentActionOnyxProps = {
+    /** ONYX PROPS */
+
+    /** The report currently being looked at */
+    report: OnyxEntry<OnyxTypes.Report>;
+
+    /** The actions from the parent report */
+    // TO DO: Replace with HOC https://github.com/Expensify/App/issues/18769.
+    parentReportActions: OnyxEntry<OnyxTypes.ReportActions>;
+};
+type ReportActionItemParentActionProps = WithLocalizeProps &
+    WindowDimensionsProps &
+    ReportActionItemParentActionOnyxProps & {
+        /** Flag to show, hide the thread divider line */
+        shouldHideThreadDividerLine: boolean;
+
+        /** Flag to display the new marker on top of the comment */
+        shouldDisplayNewMarker: boolean;
+
+        /** Position index of the report parent action in the overall report FlatList view */
+        index: number;
+
+        /** The id of the report */
+        // eslint-disable-next-line react/no-unused-prop-types
+        reportID: string;
+
+        /** The id of the parent report */
+        // eslint-disable-next-line react/no-unused-prop-types
+        parentReportID: string;
+    };
+
+function ReportActionItemParentAction({
+    report = {reportID: ''},
```

Comment:
You should be able to remove this default value completely:
```suggestion
    report,
```